docs/getting_started/installation_guide.rst

```diff
@@ -35,21 +37,19 @@ Installing OpenCMP
 
 5) Go through the `tutorials <https://opencmp.io/tutorials/index.html>`_ and other examples found in the "examples/" folder.
 
-
 Custom Commands for OpenCMP
 ===========================
 
-**Installing Optional Dependencies**
+**Using OpenCMP**
 
-* :code:`get_optional_dependencies` automatically installs all the optional dependencies.
+* :code:`opencmp` can be run using :code:`opencmp config` instead of :code:`python3 -m opencmp config`, where :code:`config` is the name of the configuration file.
 
-* :code:`get_pytest` automatically installs the :code:`pytest` python library. 
+* If providing an optional ConfigParser object for tests, please use :code:`python3 -m opencmp config ConfigParserObject` instead of :code:`opencmp config ConfigParserObject`.
```

Comment:
I don't think this is correct. How would you create the ConfigParser from the terminal since you'd be outside of the python environment.